libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
```

Comment:
Again I don't want to possibly break some devlib APIs... however, here the main goal for the regexp is to report the major.minor version and the exact SHA1 within the platform.json.

Otherwise we should update devlib to build a namedtuple with all these values...